shopfloor_delivery_shipment_mobile/static/wms/src/scenario/delivery_shipment.js

```diff
@@ -0,0 +1,445 @@
+                <v-col class="text-center" cols="12">
+                <v-btn-toggle mandatory v-model="filter_state">
+                  <v-btn color="success" value="lading">
+                    Lading
+                  </v-btn>
+                  <v-btn color="warning" value="dock">
+                    On Dock
+                  </v-btn>
+                </v-btn-toggle>
+                </v-col>
+              </v-row>
+
+            <detail-picking
+                v-if="state_is('loading_list')"
+                v-for="picking in filter_pickings(pickings())"
+                :record="picking"
+                :options="{main: true, key_title: 'name', title_action_icon: 'mdi-help-circle', on_title_action: state.onBack2Picking, fields:[{path:'carrier.name', label: 'Carrier'}, {path:'move_line_count', label:'Lines'}]}"
+                :key="make_component_key(['shipment-picking', picking.id])"
+                :card_color="getOperationColor(picking)"
+                />
+
+            <item-detail-card
+                v-if="!_.isEmpty(picking())"
+                :key="make_state_component_key(['delivery-shipment-pick', picking().id])"
+                :record="picking()"
+                :options="{main: true, key_title: 'name', title_action_field: {path: 'name', action_val_path: 'name'}}"
+                :card_color="utils.colors.color_for('screen_step_done')"
+                />
+
+            <item-detail-card
+                v-if="state_is('scan_document')"
+                :key="make_state_component_key(['delivery-shipment-shipment', shipment().id])"
+                :record="shipment()"
+                :options="{main: true, key_title: 'name', fields: [{path: 'dock.name', label: 'Dock'}]}"
+                :card_color="utils.colors.color_for('screen_step_done')"
+                />
+
+
+            <div v-if="state_is('scan_document')" v-for="(value, name, index) in this.state.data.content">
+                <v-card color="blue lighten-1" class="detail v-card mt-5 main">
+                    <v-card-title>{{ name }}</v-card-title>
+                </v-card>
+                <item-detail-card
+                    v-for="packlevel in value.package_levels"
+                    :key="make_state_component_key(['shipment-pack', packlevel.id])"
+                    :record="packlevel.package_src"
+                    :options="getPackOptions(packlevel)"
+                    :card_color="getPackColor(packlevel)"
+                    />
+                <item-detail-card
+                    v-for="line in value.move_lines"
+                    :key="make_state_component_key(['shipment-product', line.product.id])"
+                    :record="line"
+                    :options="getLineOptions(line)"
+                    :card_color="getLineColor(line)"
+                    />
+            </div>
+
+            <v-card color="warning" class="detail v-card main mb-2" v-if="state_is('validate')">
```

Comment:
this msg should come from the backend, no?

Reply:
Well it is a default message for that state that is always displayed, so maybe or maybe not :stuck_out_tongue_winking_eye: